stories/samples/general.stories.js

```diff
@@ -162,41 +162,62 @@ export const cache = () => html`
   </script>
 `;
 export const theme = () => html`
-  <div class="mgt-light">
-    <header class="mgt-dark">
+  <div>
+    <header id="dark">
       <p>I should be dark, regional class</p>
       <mgt-teams-channel-picker></mgt-teams-channel-picker>
-      <div class="mgt-light">
+      <div id="light">
         <p>I should be light, second level regional class</p>
         <mgt-teams-channel-picker></mgt-teams-channel-picker>
       </div>
     </header>
     <article>
-      <p>I should be light, global class</p>
+      <p>I should be light, global class. Toggle the color mode to see dark form</p>
```

Comment:
Is there a way for the viewer to toggle the color mode? does it make sense to allow toggling the color mode in this story?

Reply:
No color mode in this story, removing this addition.